From e6ffc08f3b01da5ca65eea419b2abbe9d03a95af Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:24 -0400
Subject: [redhat] kabi: Adding symbol fsync_bdev (fs/block_dev.c)

Message-id: <4b45d6199836517e948b5e63b911edf597cf3c3f.1410510773.git.poros@redhat.com>
Patchwork-id: 95933
O-Subject: [PATCH RHEL7.1 084/233] kabi: Adding symbol fsync_bdev (fs/block_dev.c)
Bugzilla: 1062361 1071098
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1062361,1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Write out and wait upon all dirty data associated with DMP
    device before device close. fsync_bdev() is used by VxVM
    for syncing filesystem and block device.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/fsync_bdev
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/fsync_bdev
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/fsync_bdev

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/fsync_bdev b/redhat/kabi/kabi-module/kabi_ppc64/fsync_bdev
new file mode 100644
index 0000000..b7afc5d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/fsync_bdev
@@ -0,0 +1,2 @@
+#1-
+0xa89f71e5 fsync_bdev vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/fsync_bdev b/redhat/kabi/kabi-module/kabi_s390x/fsync_bdev
new file mode 100644
index 0000000..d49859f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/fsync_bdev
@@ -0,0 +1,2 @@
+#1-
+0x1f2729a1 fsync_bdev vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/fsync_bdev b/redhat/kabi/kabi-module/kabi_x86_64/fsync_bdev
new file mode 100644
index 0000000..4ec7f5a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/fsync_bdev
@@ -0,0 +1,2 @@
+#1-
+0x8e1c90a9 fsync_bdev vmlinux EXPORT_SYMBOL
-- 
1.7.1